topics/ICRC-21/icrc_21_consent_msg.md

```diff
@@ -136,7 +136,13 @@ sequenceDiagram
 ```
 
 1. The relying party connects to the signer and requests a signature on a canister call.
-2. The signer fetches the consent message from the target canister:
+2. The chain-connected signer component fetches the consent message from the target canister:
+   * `icrc21_consent_message_request.method` must match the canister call method.
+   * `icrc21_consent_message_request.arg` must match the canister call argument.
+   * The hot signer component must use the anonymous identity for the `icrc21_consent_message` request.
```

Comment:
Side note: is "hot signer" described and referenced somewhere?

Reply:
Ah, good catch. Replaced with "chain-connected signer component" which is properly defined.